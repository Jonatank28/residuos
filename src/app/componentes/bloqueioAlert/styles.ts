import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

export const FundoModal = styled.View`
  flex: 1;
  align-items: center;
  justify-content: center;
  position: relative;
  background-color: rgba(0, 0, 0, 0.5);
`;

export const FeatherIcone = styled(FeatherIcon)``;

export const ScrollContainer = styled.ScrollView.attrs({
  vertical: true,
  showsVerticalScrollIndicator: true,
  contentContainerStyle: {
    padding: 5,
  },
})``;

export const Container = styled.View`
  width: 80%;
  height: 30%;
  justify-content: center;
  align-items: center;
  border-radius: 10px;
  background-color: ${(props) => props.theme.card.background};
`;

export const IconContainer = styled.View`
  padding-top: 20px;
  padding-bottom: 20px;
  align-items: center;
  justify-content: center;
`;

export const Modal = styled.Modal``;

export const TituloContainer = styled.View`
  flex: 0.5;
  padding-top: 5px;
  width: 100%;
  flex-direction: row;
  align-items: center;
`;

export const DescricaoContainer = styled.View`
  flex: 1;
  justify-content: center;
  width: 100%;
  padding-right: 20px;
  padding-left: 20px;
`;

export const Titulo = styled.Text`
  flex: 1;
  text-align: center;
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const Description = styled.Text`
  text-align: center;
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;
