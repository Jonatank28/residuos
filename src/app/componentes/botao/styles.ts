import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IContainer {
  backgroundColor?: string;
  hasBorder?: boolean;
  disabled?: boolean;
}


interface IText {
  textColor?: string;
}

export const Container = styled.TouchableOpacity<IContainer>`
  flex: 1;
  min-height: 50px;
  border-radius: 5px;
  position: relative;
  align-items: center;
  justify-content: center;
  border-width: ${(props) => `${props.hasBorder ? 1 : 0}px`};
  border-color: ${(props) => props.theme.border.color};
  background-color: ${(props) => props.disabled ? props.theme.border.color : props.backgroundColor ?? props.theme.button.background};
`;

export const FeatherIcone = styled(FeatherIcon)`
  position: absolute;
  top: 10px;
  left: 20px;
`;

export const TextoBotao = styled.Text<IText>`
  color: ${(props) => props.textColor ?? props.theme.text.button.color};
  font-weight: ${(props) => props.theme.text.button.fontWeight};
  font-family: ${(props) => props.theme.text.button.fontFamily};
  font-size: ${(props) => props.theme.text.button.fontSize};
`;
