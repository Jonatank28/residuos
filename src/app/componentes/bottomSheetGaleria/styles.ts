import styled from 'styled-components/native';

export const TituloSheetContainer = styled.TouchableOpacity`
  padding: 5px 0px;
  margin-bottom: 15px;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
`;

export const BottomSheetItem = styled.TouchableOpacity`
  padding: 18px;
  border-radius: 5px;
  margin-bottom: 10px;
  border-width: 1px;
  border-color: ${props => props.theme.card.border};
  background-color: ${props => props.theme.card.background};
`;

export const Titulo = styled.Text`
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const BottomSheetItemTexto = styled.Text`
  color: ${(props) => props.theme.text.body1.color};
  font-weight: ${(props) => props.theme.text.body1.fontWeight};
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;