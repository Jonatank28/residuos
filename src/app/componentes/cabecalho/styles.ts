import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';
import AwesomeIcon from 'react-native-vector-icons/FontAwesome';

interface IContainer {
  hasBorderBottom?: boolean;
  background?: string;
}

export const Container = styled.View<IContainer>`
  flex-direction: row;
  align-items: center;
  width: 100%;
  border-bottom-color: ${(props) => props.theme.border.color};
  border-bottom-width: ${(props) => (props.hasBorderBottom ? '1px' : '0px')};
  background-color: ${(props) => props.background ?? props.theme.primary};
`;

export const OfflineContainer = styled.View`
  height: 4px;
  background-color: ${(props) => props.theme.colors.orange};
`;

export const FeatherIcone = styled(FeatherIcon)``;

export const AwesomeIcone = styled(AwesomeIcon)``;

export const BotaoEsquerdoContainer = styled.TouchableOpacity`
  flex: 0.2;
  padding: 15px 0px 15px 0px;
  align-items: center;
`;

export const TituloContainer = styled.View`
  flex: 1;
`;

export const Titulo = styled.Text`
  text-align: center;
  color: ${(props) => props.theme.secundary};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const BotaoDireitoContainer = styled.View`
  flex: 0.2;
  padding: 15px 0px 15px 0px;
  align-items: center;
`;
