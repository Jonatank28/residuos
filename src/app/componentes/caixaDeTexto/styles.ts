import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IContainer {
  temErro: boolean;
  editavel?: boolean;
  backgroundColor?: string;
  borderLeftUp: number;
  borderRightUp: number;
  borderLeftDown: number;
  borderRightDown: number;
  marginTop: number;
  marginBottom: number;
  marginRight: number;
  marginLeft: number;
}

export const Container = styled.View<IContainer>`
  flex: 1;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  min-height: 50px;
  padding-left: 10px;
  border-top-left-radius: ${(props) => `${props.borderLeftUp}px`};
  border-top-right-radius: ${(props) => `${props.borderRightUp}px`};
  border-bottom-left-radius: ${(props) => `${props.borderLeftDown}px`};
  border-bottom-right-radius: ${(props) => `${props.borderRightDown}px`};
  border-color: ${(props) => props.theme.colors.accent};
  border-width: ${(props) => `${props.temErro ? 1 : 0}px`};
  margin-top: ${(props) => `${props.marginTop}px`};
  margin-bottom: ${(props) => `${props.marginBottom}px`};
  margin-right: ${(props) => `${props.marginRight}px`};
  margin-left: ${(props) => `${props.marginLeft}px`};
  border-width: 1px;
  border-color: ${(props) => props.theme.border.color};
  background-color: ${(props) => !props.editavel ? '#F1f1f1' : props.backgroundColor ?? props.theme.card.background};
`;

export const FeatherIcone = styled(FeatherIcon)``;

export const Input = styled.TextInput`
  flex: 1;
  height: 40px;
  padding-left: 10px;
  width: 100%;
  color: ${(props) => props.theme.text.input.color};
  /* background-color: ${(props) => !props.editable ? props.theme.card.border : props.theme.text.input.background}; */
`;

export const ErroFormContainer = styled.View`
  align-items: flex-end;
  padding-right: 8px;
  margin-top: 5px;
`;

export const ErroFormTexto = styled.Text`
  color: ${(props) => props.theme.colors.accent};
  font-weight: ${(props) => props.theme.text.caption.fontWeight};
  font-family: ${(props) => props.theme.text.caption.fontFamily};
  font-size: ${(props) => props.theme.text.caption.fontSize};
`;
