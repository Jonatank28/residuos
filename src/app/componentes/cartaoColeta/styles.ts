import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IContainer {
  isDetails?: boolean;
}

interface IImage {
  width?: number;
  height?: number;
}

interface IRowContainer {
  flex?: number;
  isBottom?: boolean;
  justifyContent?: string;
  marginTop?: number;
  marginBottom?: number;
  marginLeft?: number;
  marginRight?: number;
}

interface IOpcaoContainer {
  backgroundColor?: string;
  hasBorder?: boolean;
}

interface ITitulo {
  color?: string;
  fontSize?: number;
}

export const Container = styled.TouchableOpacity<IContainer>`
  flex-direction: row;
  margin-bottom: 10px;
  border-radius: ${props => `${props.isDetails ? 5 : 0}px`};
  background-color: ${props => props.theme.card.background};
`;

export const FeatherIcone = styled(FeatherIcon)`
  color: ${props => props.theme.secundary};
`;

export const StatusTextoContainer = styled.View`
  align-items: flex-end;
  padding-bottom: 10px;
  padding-right: 10px;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding-left: 10;
`;

export const OpcaoContainer = styled.TouchableOpacity<IOpcaoContainer>`
  padding: 5px;
  margin: 0px 2px;
  border-radius: 5px;
  max-height: 25px;
  align-items: center;
  justify-content: center;
  background-color: ${props => props.backgroundColor ?? props.theme.colors.orange};
`;

export const StatusContainer = styled.View<IOpcaoContainer>`
  width: 10px;
  border-top-left-radius: ${props => `${props.hasBorder ? 5 : 0}px`};
  border-bottom-left-radius: ${props => `${props.hasBorder ? 5 : 0}px`};
  background-color: ${props => props.backgroundColor ?? props.theme.colors.green};
`;

export const ConteudoContainer = styled.View`
  flex: 1;
`;

export const Image = styled.Image`
  width: 100%;
  height: 100%;
`;

export const ImageContainer = styled.View<IImage>`
  width: ${props => `${props.width ?? 50}px`};
  height: ${props => `${props.height ?? 50}px`};
  margin-right: 5px;
`;

export const TituloContainer = styled.View`
  flex: 1;
`;

export const DescricaoPendenteContainer = styled.View`
  justify-content: center;
`;

export const DataContainer = styled.View`
  flex: 1;
  align-items: flex-end;
`;

export const RowContainer = styled.View<IRowContainer>`
  flex: ${props => props.flex ?? 1};
  justify-content: ${props => props.justifyContent ?? 'flex-start'};
  padding-right: ${props => `${props.marginRight ?? 10}px`};
  padding-left: ${props => `${props.marginLeft ?? 10}px`};
  padding-top: ${props => `${props.marginTop ?? 10}px`};
  padding-bottom: ${props => `${props.marginBottom ?? 10}px`};
  flex-direction: row;
  align-items: ${props => (props.isBottom ? 'center' : 'flex-start')};
`;

export const DescricaoTextoContainer = styled.View`
  flex: 1;
  flex-direction: row;
`;

export const RoteirizacaoContainer = styled.View`
  flex: 1;
  padding-left: 10px;
  padding-bottom: 5px;
`;

export const ColumnContainer = styled.View`
  flex: 1;
  padding: 0px 10px;
`;

export const HoraTexto = styled.Text<ITitulo>`
  color: ${props => props.theme.text.subhead.color};
  font-weight: ${props => props.theme.text.subhead.fontWeight};
  font-family: ${props => props.theme.text.subhead.fontFamily};
  font-size: ${props => (props.fontSize ? `${props.fontSize}px` : props.theme.text.subhead.fontSize)};
`;

export const DescricaoPendente = styled.Text<ITitulo>`
  text-transform: uppercase;
  color: ${props => props.color ?? props.theme.colors.accent};
  font-weight: ${props => props.theme.text.caption.fontWeight};
  font-family: ${props => props.theme.text.caption.fontFamily};
  font-size: ${props => props.theme.text.caption.fontSize};
`;

export const RoteirizacaoTexto = styled.Text`
  color: ${props => props.theme.primary};
  font-weight: ${props => props.theme.text.body1.fontWeight};
  font-family: ${props => props.theme.text.body1.fontFamily};
  font-size: ${props => props.theme.text.body1.fontSize};
`;

export const DescricaoTexto = styled.Text`
  color: ${props => props.theme.text.body1.color};
  font-weight: ${props => props.theme.text.body1.fontWeight};
  font-family: ${props => props.theme.text.body1.fontFamily};
  font-size: ${props => props.theme.text.body1.fontSize};
`;

export const EnderecoContainer = styled.View`
  flex: 1;
`;

export const OpcoesContainer = styled.View`
  flex-direction: row;
  z-index: 999;
`;

export const DescricaoTextoTitulo = styled.Text`
  padding-right: 5px;
  color: ${props => props.theme.text.body1.color};
  font-weight: 700;
  font-family: ${props => props.theme.text.body1.fontFamily};
  font-size: ${props => props.theme.text.body1.fontSize};
`;

export const TextoRowContainer = styled.View`
  flex-direction: row;
`;

export const OSTexto = styled.Text<ITitulo>`
  color: ${props => props.theme.text.headline.color};
  font-weight: ${props => props.theme.text.headline.fontWeight};
  font-family: ${props => props.theme.text.headline.fontFamily};
  font-size: ${props => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;

export const Titulo = styled.Text<ITitulo>`
  color: ${props => props.color ?? props.theme.text.headline.color};
  font-weight: ${props => props.theme.text.headline.fontWeight};
  font-family: ${props => props.theme.text.headline.fontFamily};
  font-size: ${props => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;
