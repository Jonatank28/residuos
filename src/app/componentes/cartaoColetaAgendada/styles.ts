import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IRowContainer {
  flex?: number;
  isBottom?: boolean;
  justifyContent?: string;
  marginTop?: number;
  marginBottom?: number;
  marginLeft?: number;
  marginRight?: number;
}

interface IOpcaoContainer {
  backgroundColor?: string;
}

interface ITitulo {
  color?: string;
  fontSize?: number;
}

export const Container = styled.TouchableOpacity`
  flex-direction: row;
  background-color: ${(props) => props.theme.card.background};
`;

export const FeatherIcone = styled(FeatherIcon)`
  color: ${(props) => props.theme.secundary};
`;

export const OpcaoContainer = styled.TouchableOpacity<IOpcaoContainer>`
  padding: 5px;
  margin: 0px 2px;
  border-radius: 5px;
  max-height: 25px;
  align-items: center;
  justify-content: center;
  background-color: ${(props) => props.backgroundColor ?? props.theme.colors.orange};
`;

export const StatusContainer = styled.View`
  width: 10px;
  background-color: ${(props) => props.theme.primary};
`;

export const ConteudoContainer = styled.View`
  flex: 1;
`;

export const PeriodicidadeContainer = styled.View`
  position: absolute;
  top: -10px;
  right: 10px;
  padding: 2px 4px;
  border-radius: 25px;
  background-color: ${(props) => props.theme.colors.green};
`;

export const Image = styled.Image`
  width: 100%;
  height: 100%;
`;

export const ImageContainer = styled.View`
  width: 45px;
  height: 45px;
  margin-right: 5px;
`;

export const TituloContainer = styled.View`
  flex: 1;
`;

export const DataContainer = styled.View`
  flex: 1;
  align-items: flex-end;
`;

export const RowContainer = styled.View<IRowContainer>`
  flex: ${(props) => props.flex ?? 1};
  justify-content: ${(props) => props.justifyContent ?? 'flex-start'};
  padding-right: ${(props) => `${props.marginRight ?? 10}px`};
  padding-left: ${(props) => `${props.marginLeft ?? 10}px`};
  padding-top: ${(props) => `${props.marginTop ?? 10}px`};
  padding-bottom: ${(props) => `${props.marginBottom ?? 10}px`};
  flex-direction: row;
  align-items: ${(props) => (props.isBottom ? 'center' : 'flex-start')};
`;

export const DescricaoTextoContainer = styled.View`
  flex: 1;
  flex-direction: row;
`;

export const RoteirizacaoContainer = styled.View`
  flex: 1;
  padding-left: 10px;
  padding-bottom: 5px;
`;

export const ColumnContainer = styled.View`
  flex: 1;
  padding: 0px 10px;
`;

export const HoraTexto = styled.Text<ITitulo>`
  color: ${(props) => props.theme.text.subhead.color};
  font-weight: ${(props) => props.theme.text.subhead.fontWeight};
  font-family: ${(props) => props.theme.text.subhead.fontFamily};
  font-size: ${(props) => (props.fontSize ? `${props.fontSize}px` : props.theme.text.subhead.fontSize)};
`;

export const RoteirizacaoTexto = styled.Text`
  color: ${(props) => props.theme.primary};
  font-weight: ${(props) => props.theme.text.body1.fontWeight};
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;

export const DescricaoTexto = styled.Text`
  color: ${(props) => props.theme.text.body1.color};
  font-weight: ${(props) => props.theme.text.body1.fontWeight};
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;

export const EnderecoContainer = styled.View`
  flex: 1;
`;

export const OpcoesContainer = styled.View`
  flex-direction: row;
  z-index: 999;
`;

export const DescricaoTextoTitulo = styled.Text`
  padding-right: 5px;
  color: ${(props) => props.theme.text.body1.color};
  font-weight: 700;
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;

export const TextoRowContainer = styled.View`
  flex-direction: row;
`;

export const OSTexto = styled.Text<ITitulo>`
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;

export const Titulo = styled.Text<ITitulo>`
  color: ${(props) => props.color ?? props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;

export const PeriodicidadeTexto = styled.Text`
  color: ${(props) => props.theme.colors.white};
  font-weight: ${(props) => props.theme.text.caption.fontWeight};
  font-family: ${(props) => props.theme.text.caption.fontFamily};
  font-size: 10px;
`;
