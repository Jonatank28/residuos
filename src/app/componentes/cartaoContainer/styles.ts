import styled from 'styled-components/native';

export const Container = styled.View`
  margin-bottom: 10px;
  padding: 10px;
  border-radius: 5px;
  background-color: ${(props) => props.theme.card.background};
`;

export const RowContainer = styled.View`
  flex: 1;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
`;

export const TituloContainer = styled.View`
  padding: 5px 10px;
`;

export const DetalhesContainer = styled.View`
  padding: 5px;
`;

export const Titulo = styled.Text`
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const Descricao = styled.Text`
  color: ${(props) => props.theme.text.body2.color};
  font-weight: ${(props) => props.theme.text.body2.fontWeight};
  font-family: ${(props) => props.theme.text.body2.fontFamily};
  font-size: ${(props) => props.theme.text.body2.fontSize};
`;
