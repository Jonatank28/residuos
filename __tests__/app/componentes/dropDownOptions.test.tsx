import React from 'react';
import renderer from 'react-test-renderer';

import { ThemeProvider } from 'styled-components/native';
import * as Themes from '../../../src/app/styles/themes';
import DropDownOptions, { IDropDownItems } from '../../../src/app/componentes/dropdownOptions';

describe('<DropDownOptions />', () => {
  beforeEach(() => {
    jest.resetAllMocks();
    jest.useFakeTimers();
  });

  afterEach(() => {
    jest.useRealTimers();
  });

  it('matches last snapshot', async () => {
    const items: IDropDownItems[] = [
      { label: '1', value: 'test 1' },
      { label: '2', value: 'test 2' },
    ];

    const tree: any = await renderer.create(
      <ThemeProvider theme={Themes.light}>
        <DropDownOptions
          items={items}
          value="test 1"
          onChange={() => {}}        />
      </ThemeProvider>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
    expect(tree.children.length).toBe(2);
  });
});
