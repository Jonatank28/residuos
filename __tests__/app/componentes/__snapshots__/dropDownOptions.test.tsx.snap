// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropDownOptions /> matches last snapshot 1`] = `
<View
  style={
    Array [
      Object {},
      Object {
        "zIndex": 5000,
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onLayout={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "borderBottomLeftRadius": 5,
        "borderBottomRightRadius": 5,
        "borderColor": "black",
        "borderStyle": "solid",
        "borderTopLeftRadius": 5,
        "borderTopRightRadius": 5,
        "borderWidth": 0,
        "flex": 1,
        "flexDirection": "row",
        "marginBottom": 45,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "minHeight": 50,
        "opacity": 1,
        "paddingHorizontal": 10,
        "paddingVertical": 5,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#000",
          },
          Object {},
          Object {
            "flex": 1,
            "opacity": 1,
          },
          Object {
            "marginLeft": 5,
            "marginRight": 0,
          },
          [Function],
          Object {
            "color": "#3C3C3C",
            "flex": 0.98,
            "marginLeft": 10,
            "marginRight": 10,
          },
          false,
          Object {},
        ]
      }
    >
      
    </Text>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "borderBottomLeftRadius": 0,
            "borderTopLeftRadius": 0,
            "flexDirection": "row",
            "justifyContent": "center",
            "paddingVertical": 8,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {},
            Object {
              "opacity": 1,
            },
          ]
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#000",
                "fontSize": 25,
              },
              undefined,
              Object {
                "fontFamily": "Feather",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          ï„®
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderBottomLeftRadius": 5,
          "borderBottomRightRadius": 5,
          "borderColor": "#dfdfdf",
          "borderTopLeftRadius": 5,
          "borderTopRightRadius": 5,
          "borderWidth": 1,
          "paddingHorizontal": 10,
          "paddingVertical": 5,
        },
        Object {
          "alignItems": "center",
          "borderTopLeftRadius": 0,
          "borderTopRightRadius": 0,
          "justifyContent": "center",
          "position": "absolute",
          "width": "100%",
        },
        Object {
          "backgroundColor": "#FFFFFF",
        },
        Object {
          "borderWidth": 0,
          "display": "none",
          "position": "relative",
        },
        Object {
          "maxHeight": 150,
          "top": undefined,
          "zIndex": 5000,
        },
      ]
    }
  >
    <RCTScrollView
      nestedScrollEnabled={true}
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <View>
        <View
          onLayout={[Function]}
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "borderBottomColor": "#F5F4F4",
                "borderBottomWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "minHeight": 50,
                "opacity": 1,
                "paddingVertical": 8,
                "width": "100%",
              }
            }
          >
            <View
              style={
                Object {
                  "alignContent": "center",
                  "flexDirection": "row",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {},
                    Object {
                      "color": "#3C3C3C",
                      "flex": 0.98,
                      "marginLeft": 10,
                      "marginRight": 10,
                    },
                    false,
                    Object {},
                  ]
                }
              >
                1
              </Text>
            </View>
          </View>
        </View>
        <View
          onLayout={[Function]}
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "borderBottomColor": "#F5F4F4",
                "borderBottomWidth": 1,
                "flexDirection": "row",
                "justifyContent": "center",
                "minHeight": 50,
                "opacity": 1,
                "paddingVertical": 8,
                "width": "100%",
              }
            }
          >
            <View
              style={
                Object {
                  "alignContent": "center",
                  "flexDirection": "row",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {},
                    Object {
                      "color": "#3C3C3C",
                      "flex": 0.98,
                      "marginLeft": 10,
                      "marginRight": 10,
                    },
                    false,
                    Object {},
                  ]
                }
              >
                2
              </Text>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
